moveit_ros_control_interface/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>moveit_ros_control_interface</name>
+  <version>0.0.0</version>
```

Comment:
has to actually match the other package.xml's in the same repo (0.5.6) in order to be released